### ISSUE TITLE
Stuck on Injected, never shows connected, and no packets are sniffed

### ISSUE THREAD

**Issue by naeem76** [frostycpu/FinalesFunkeln#10]

**Comment by frostycpu** [comment]
Region?

**Comment by naeem76** [comment]
EUW

**Comment by frostycpu** [comment]
EUW works fine for me. Are you running it as an admin? Did you try uninstalling an reinstalling the euw certificate?

**Comment by naeem76** [comment]
Yup, as admin always, I'm currently using your certificate on LoLnotes which is working, so certificate isn't the issue I guess.

**Comment by frostycpu** [comment]
Does it work for any other region?

**Comment by naeem76** [comment]
Um do you have the cert for SG MY server in garena?

**Comment by DantesJ805** [comment]
I'm getting this issue too. Does this still work for the current client? 11.10
